client/webserver/site/src/js/wallets.ts

```diff
@@ -743,10 +742,15 @@ export default class WalletsPage extends BasePage {
 
       if (spot) {
         const convRate = app().conventionalRate(baseid, quoteid, spot.rate, exchanges[host])
-        tmpl.price.textContent = fourSigFigs(convRate)
+        tmpl.price.textContent = withNumDecimals(convRate, mkt.convRateNumDecimals)
```

Comment:
I think the point is to lop off extra zeros in the decimals. In many cases, this will force the showing of decimal places we don't care about. For example, if the current rate is `1,284.0`, and the rate step is `1e4`, this will show `1,284.0000`, which is clearly not what we want. Sig figs is the way to go. We just need to clean up the max number of decimals too.

Reply:
You're right, no need for exact values.

Reply:
Isn't the intent then to be using {min,max}imum**Fraction**Digits, not {min,max}imum**Significant**Digits?

Also, regardless of value, I feel like we'd want at least one decimal e.g. `1,284.0` is Buck's example.

Reply:
For instance:

```js
const FourSigFigsSmall = new Intl.NumberFormat('en-US', {
  minimumFractionDigits: 1,
  maximumSignificantDigits: 5
})

const FourSigFigsBig = new Intl.NumberFormat('en-US', {
  minimumFractionDigits: 1,
  maximumFractionDigits: 1
})

function fourSigFigs (v) {
  if (v<1000) return FourSigFigsSmall.format(v)
  return FourSigFigsBig.format(v)
}

console.log(fourSigFigs(123456.789))
console.log(fourSigFigs(1234.789))
console.log(fourSigFigs(123.789))
console.log(fourSigFigs(0.01357))
```

Output:

```
> "123,456.8"
> "1,234.8"
> "123.79"
> "0.01357"
```